### DISCUSSION TITLE
Some in-depth component examples

### DISCUSSION THREAD

**Discussion by TheMouster** [defx/synergy#8] [Ideas]
Hi,

If it’s possible some in-depth examples would be really helpful. I really like synergy and I think its’ got some very good benefits personally. However my JavaScript is a bit on the lack-lustre side, especially when it gets to navigating object heirarchies, so some component examples would be very helpful. Especially relating to components that might render a repeated block, so that I can get a better handle on how they work.

Thanks

**Comment by defx** [comment]
Hi @TheMouster,

Yes, I couldn't agree more and that's something I've been working on whenever possible. I've literally just updated the docs at synergyjs.org to include a "learn" section. It only has one very simple example so far, but now there's a place for those things I'm going to aim to add at least one new example each week until there's a decent amount to help folks get started. Now that you've mentioned it I'll be sure to get one in for repeated blocks. :)

**Reply by TheMouster** [reply]
Cheers! Thanks for all the hard work. I’ve been looking at the competition over the past few days and I’ve yet to find anything that works as simply and easily. Closest was ottavino but that’s not as simple from a usability perspective. I also quite like the look of Microsofts FAST too, but it’s all typescript and, whilst I like typescript, it’s just yet another burden from a development point-of-view. FYI: I’m a vanilla JS fan, or as close to it as I can get without being entirely stupid about it, which is where synergy fits in so well.

**Reply by defx** [reply]
That's really great to hear. At this point in time with next to no community it makes such a huge difference to have somebody asking questions and giving feedback so thank you!
